README.md

```diff
@@ -38,6 +38,12 @@ This Pipedrive API client is distributed under the MIT licence.
 - Run unit and integration tests
 - Make PR
 
+## Options
+* `strictMode` - In strict mode `*_id` items in the responses are numeric IDs. Default is *false* in which case expanded
+ objects are returned. Strict mode is recommended and is likely to be the default in the future.
+* `oauth` - whether the API token is to be used as OAuth bearer token instead of classic API key (default is *false*). 
+When setting `oauth` to true your application code must take care of fetching, storing and refershing the tokens.
```

Comment:
> must take care of fetching, storing and refershing the tokens
maybe add example how to have these interactions?

Reply:
@tot-ra Those examples would be a lot of work to add. I added this comment so it would be clear that this PR only changes the way requests are made with this SDK and is not a full blown implementation of OAuth.